playground/recipe_opstack.go

```diff
@@ -96,21 +101,38 @@ func (o *OpRecipe) Apply(ctx *ExContext, artifacts *Artifacts) *Manifest {
 		flashblocksBuilderURLRef = ConnectWs("op-rbuilder", "flashblocks")
 	}
 
+	if o.flashblocks {
+		peers = append(peers, "flashblocks-rpc")
+	}
+	
+	svcManager.AddService("bproxy", &BProxy{
+		TargetAuthrpc: externalBuilderRef,
+		Peers: peers,
+		Flashblocks: o.flashblocks,
+		FlashblocksBuilderURL: flashblocksBuilderURLRef,
+	})
+
+	svcManager.AddService("websocket-proxy", &WebsocketProxy{
+		Upstream: "rollup-boost",
+	})
+
 	elNode := "op-geth"
 	if o.externalBuilder != "" {
 		elNode = "rollup-boost"
 
 		svcManager.AddService("rollup-boost", &RollupBoost{
 			ELNode:                "op-geth",
-			Builder:               externalBuilderRef,
+			Builder:               Connect("bproxy", "authrpc"),
```

Comment:
can bproxy be enabled / disabled via a flag? as it is specific to our infra

Reply:
It's improssible to add additional ELs without some kind of mirroring proxy, but i could make bproxy used only if we have additional ELs